tests/plugins/test_pulp_pull.py

```diff
@@ -49,6 +49,15 @@ def workflow(self, push=True, sync=True, build_process_failed=False):
         if sync:
             push_conf.add_pulp_registry('pulp', crane_uri=self.CRANE_URI, server_side_sync=True)
 
+        def custom_retries(*args, **kwargs):
+            kwargs['times'] = 0
+            return retry_fnc(*args, **kwargs)
+
+        retry_fnc = atomic_reactor.util.do_retries
+        (flexmock(atomic_reactor.util)
+            .should_receive('do_retries')
+            .replace_with(custom_retries))
```

Comment:
This is repeated a lot. I wonder if it should be somewhere common.

Reply:
added to retry_mock